test/puppetlabs/trapperkeeper/services/authorization/authorization_service_test.clj

```diff
@@ -89,9 +137,22 @@
             {:keys [status body]} (app req)]
         (is (= status 200))
         (is (= body "Prefix: !dlroW olleH")))))
+  (testing "(TK-260) Authorizing unauthenticated requests"
```

Comment:
Might also be good to add a test which has an unauthenticated request being made to a URI which does match a rule but where the rule has been configured with `allow_unauthenticated: false` (explicitly or implicitly by not having `allow_unauthenticated` specified) to confirm that the request is still denied.